README.md

```diff
@@ -168,20 +168,20 @@ Code snippets like `var foo = "bar";` can be shown inline.
 Also, `this should vertically align` ~~`with this`~~ ~~and this~~.
 
 Code can also be shown in a block element.
-````
+```
```

Comment:
Yes, it's good to have a blank line there, but I guess that's a style preference.

For example the markdown syntax highlighting implementation that I use in Sublime doesn't work if I omit the blank line before the code block.